docs/rhoas/rhoas-service-contexts/README.adoc

```diff
@@ -275,14 +275,28 @@ $ rhoas context use --name development-context
 $ cd ~/app-services-guides/code-examples/quarkus-service-registry-quickstart/
 ----
 
+. Create a service account for the Quarkus application to authenticate with the Kafka and {registry} instances in the context. Save the credentials in an environment variables file in the directory for the producer component.
++
+[source,shell]
+----
+$ rhoas service-account create --type env --output-file ./producer/.env
+----
+
```

Comment:
@rkpattnaik780 Is there a workflow to specify an existing service account, if you don't create a new one? I think what we have here is OK for v1 of these updates, but if users can also choose an existing service account, then I'll create a follow-up issue to test and document those additional couple of steps.

Reply:
If users want to choose an existing service account, they will need to have the credentials in a `.env` file and use it instead.
It might get tricky, if the user has the credentials saved in a json, the format will need to be manually changed. I think the workflow can be different according to what users may prefer, so I am not sure if we should document it.

Reply:
That makes sense, @rkpattnaik780. Maybe we'll continue to think about what we might document in that regard - even if it's just a note. In the meantime,  I don't think it should block us from merging this PR.